exporter/awsemfexporter/emf_exporter_test.go

```diff
@@ -49,17 +51,24 @@ type mockPusher struct {
 func (p *mockPusher) AddLogEntry(logEvent *LogEvent) error {
 	args := p.Called(nil)
 	errorStr := args.String(0)
+	if errorStr == "expired token" {
```

Comment:
Is there any documentation for this string? If it's the best we've got it's better than nothing but ideally we have a less fragile-seeming check